content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+```
+
+❗Note: Before using the AWS CLI, remember to export the environment AWS\_CA\_BUNDLE=/opt/mapr/conf/ca/chain-ca.pem.
+Otherwise AWS CLI cannot communicate with S3server because S3server is using self-signed TLS certificates.
+
+After inputting the above command, the AWS CLI will ask you to input the access key and secret key.
+After the profile is created, the information will be stored at <ins>$HOME/.aws/config</ins> and <ins>$HOME/.aws/credentials</ins>.
+
+Use the below command to list Buckets:
+
+```shell
+aws s3api list-buckets --endpoint-url https://`hostname -f`:9000 --profile s3-test-iam_user-ray-2-objstor
+```
+
+Use the below command to put a file into the Bucket:
+
+```shell
+aws s3api put-object --bucket s3-test-iam-user-bucket --key 'testdir/s3-test-iam-user-dir/hpe-cp-rhel-release-5.5.1-3083.bin' --body 'downloads/hpe-cp-rhel-release-5.5.1-3083.bin' --endpoint-url https://m2-maprts-vm197-172.mip.storage.hpecorp.net:9000 --profile s3-test-iam_user-ray-2-objstor
+```
+
+🗒Note: "s3-test-iam-user-bucket" is the Bucket's name that you created before.
+"testdir/s3-test-iam-user-dir/hpe-cp-rhel-release-5.5.1-3083.bin" is the path that you want to put into the Bucket.
+The part of "testdir/s3-test-iam-user-dir/" indicates it's under this directory, if the directory doesn't exist, it will be created.
+"downloads/hpe-cp-rhel-release-5.5.1-3083.bin" is the local file path that you want to put into the Bucket.
+
+## Create an Cold-Tiered Volume and offlad to remote Object Store
+
+So, now we are going to create a Volume on another cluster and configure the Cold Tier remote target for this Volume.
+Then we will manually offload the data in this Volume to the remote HPE Ezmeral Data Fabric Object Store.
+
+### Create a Cold-Tiered Volume by Web GUI
+
+First of all, you need to log in to MCS, and enter the following positions in turn at the top of the screen: Data --> Volumes👇
+
+<a href="https://ibb.co/1Zg5RXN"><img src="https://i.ibb.co/2q9QcMr/Create-Cold-Tier-Volume-1.png" alt="Create-Cold-Tier-Volume-1" border="0"></a>
+
+Then click "Create Volume" at the top of the screen👇.
+
+<a href="https://ibb.co/T8SV1Mt"><img src="https://i.ibb.co/cFHGwhQ/Create-Cold-Tier-Volume-2.png" alt="Create-Cold-Tier-Volume-2" border="0"></a>
+
+Fill in the necessary information, you can refer to this document 👉 [Creating a Volume](https://docs.datafabric.hpe.com/72/ClusterAdministration/data/volumes/CreateVols.html).
+
+Turn on the "Data Tiering" switch and select "Remote Archiving(Cold)". Refer to the figure below to fill in the remote target information:
+
+<a href="https://ibb.co/cyy3cWt"><img src="https://i.ibb.co/3TTdyxW/Create-Cold-Tier-Volume-3.png" alt="Create-Cold-Tier-Volume-3" border="0"></a>
+
+* URL: The host where the S3server of the remote HPE Ezmeral Data Fabric cluster is located, and the port number is the default port 9000 of the S3server.
+* Bucket: The Bucket created for IAM User in previous steps.
```

Comment:
This would change to read "Bucket: The bucket created for IAM User in previous steps."